backend/src/main/java/aiversum/backend/rest/ChatController.java

```diff
@@ -36,6 +36,12 @@ public Flux<String> generate(@PathVariable String provider, @RequestBody String
                     .body(Mono.just(chatCommand), String.class)
                     .retrieve()
                     .bodyToFlux(String.class);
+            case "genai" -> webClient.post()
+                    .uri(STR."https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash-latest:generateContent?key=\{propertiesConfig.genai().GOOGLE_API_KEY()}")
```

Comment:
Please provide an option to pick the model

Reply:
might be "gemini"?